src/check_datapackage/internals.py

```diff
@@ -26,6 +26,12 @@ def _get_fields_at_jsonpath(
     return _map(matches, _create_descriptor_field)
 
 
+def _get_direct_jsonpaths(jsonpath: str, descriptor: dict[str, Any]) -> list[str]:
```

Comment:
Should this `descriptor` argument be renamed then if it is being used for things that aren't `descriptor`s?